src/scripts/main.js

```diff
@@ -1,3 +1,48 @@
 'use strict';
 
-// write your code here
+let bodyDoc;
+let mateLogo;
+
+if (document.querySelector('body') !== null) {
+  bodyDoc = document.querySelector('body');
+}
+
+if (document.querySelector('.logo') !== null) {
+  mateLogo = document.querySelector('.logo');
+}
+
+const promise1 = () => {
+  return new Promise((resolve, reject) => {
+    mateLogo.addEventListener('click', () => resolve('Promise was resolved!'), {
+      once: true,
+    });
+  });
+};
+
+const promise2 = () => {
+  return new Promise((resolve, reject) => {
+    // eslint-disable-next-line prefer-promise-reject-errors
+    setTimeout(() => reject('Promise was rejected!'), 3000);
```

Comment:
The 'reject' function of the 'promise2' is not being used. If you do not intend to handle a rejection condition, consider removing the unused parameter to avoid confusion.